src/basic/meson.build

```diff
@@ -16,6 +16,7 @@ basic_sources = files(
         'chase.c',
         'chattr-util.c',
         'conf-files.c',
+        'conf-virt.c',
```

Comment:
"conf" is usually used for "configuration" (as above conf-files) so please spell it out as "confidential" to avoid confusion